### DISCUSSION TITLE
Does rose-pine have a color requirement like 256 or true colors?

### DISCUSSION THREAD

**Discussion by dawilliams** [rose-pine/neovim#158] [Q&A] [answered]
I'm trying to use rose-pine with neovim 0.9.0 and tree-sitter on the default macOS terminal app. The terminal is set to xterm-256color but the colors are way off. For example parentheses and empty lines have a teal background. My first guess is that the default mac terminal can't support the color requirements.

**Comment by mvllow** [comment, marked as answer]
You are correct, unfortunately termimal.app does not support true colour. I would personally recommend kitty terminal as an alternative 💜
